docs/src/content/docs/id/guides/site-search.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Pencarian Situs
+description: Pelajari tentang fitur pencarian situs bawaan Starlight dan cara mengkustomisasinya.
+sidebar:
+  badge: Baru
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Secara default, situs Starlight termasuk pencarian teks lengkap yang didukung oleh [Pagefind](https://pagefind.app/), yang merupakan alat pencarian cepat dan bandwidth rendah untuk situs statis.
+
+Tidak diperlukan konfigurasi untuk mengaktifkan pencarian. Bangun dan terapkan situs Anda, kemudian gunakan bilah pencarian di header situs untuk menemukan konten.
+
+Kecualikan sebuah halaman
+
+## Sembunyikan konten dalam hasil pencarian
+
+### Kecualikan sebuah halaman
+
+Untuk mengecualikan sebuah halaman dari indeks pencarian Anda, tambahkan [`pagefind: false`](/id/reference/frontmatter/#pagefind) ke frontmatter halaman:
+
+```md title="src/content/docs/not-indexed.md" ins={3}
+---
+title: Content to hide from search
```

Comment:
The title could be translated to give more context to the reader (and technically the `not-indexed` part of the filename too but it depends if this makes sense for Indonesian or not, if this would help or not, so I'll let you decide).